src/audio/dai.c

```diff
@@ -64,8 +64,7 @@ struct dai_data {
 	int xrun;		/* true if we are doing xrun recovery */
 
 	/* processing function */
-	void (*process)(struct comp_buffer *source, struct comp_buffer *sink,
-			uint32_t bytes);
+	pcm_converter process;
```

Comment:
can you comment this like some of the other members

Reply:
done